scripts/release.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env sh
+set -e
+PATH=$PATH:$PWD/bin
+GOPATH=$PWD
+
+TAG=$( cd src/github.com/SUSE/cf-plugin-backup
+       git describe --tags --long | awk -F - '{ print $1 }' )
+
+NAME=$( cd src/github.com/SUSE/cf-plugin-backup
+        basename $(git config --get remote.origin.url) .git)
+
+# Assemble configuration data for `out` to git-release resource.
+echo > release/name ${NAME}
+echo > release/tag  ${TAG}
+echo > release/body "${NAME}: Release ${TAG}"
+
+# Build the tarballs for the release
+make -C src/github.com/SUSE/cf-plugin-backup build
+make -C src/github.com/SUSE/cf-plugin-backup dist
+
+# Show configuration in log
+echo Release information:
+for path in release/*
+do
+    cat $path | sed -e "s/^/$(basename $path):	/"
```

Comment:
```bash
printf "%s: " "${path#**/}"
cat $path
```

Reply:
This changes the output. My code places filename and content on a single line.
This code places them on separate lines.

Reply:
Printf is sneaky and _doesn't_ append a new line on the output :)

Reply:
Oh. Ok then, will make the change.